metricbeat/module/system/process/process.go

```diff
@@ -107,6 +125,27 @@ func (m *MetricSet) Fetch(r mb.ReporterV2) error {
 	}
 
 	for evtI := range procs {
+		if m.networkMonitoring != nil {
+			pid, err := extractPid(roots[evtI])
+			if err != nil {
+				m.Logger().Debugf("error fetching pid for network data: %s", err)
```

Comment:
A lot of the system/process error messages are at debug level, since they're fairly easy to break for relatively non-fatal errors or config issues. Decided to do the same here.